packages/core-data/src/selectors.js

```diff
@@ -4,77 +4,6 @@
 import createSelector from 'rememo';
 import { map, find, get, filter } from 'lodash';
 
-/**
- * WordPress dependencies
- */
-import { select } from '@wordpress/data';
-
-/**
- * Internal dependencies
- */
-import { REDUCER_KEY } from './name';
-
-/**
- * Returns true if resolution is in progress for the core selector of the given
- * name and arguments.
- *
- * @param {string} selectorName Core data selector name.
- * @param {...*}   args         Arguments passed to selector.
- *
- * @return {boolean} Whether resolution is in progress.
- */
-function isResolving( selectorName, ...args ) {
-	return select( 'core/data' ).isResolving( REDUCER_KEY, selectorName, ...args );
-}
-
-/**
- * Returns all the available terms for the given taxonomy.
- *
- * @param {Object} state    Data state.
- * @param {string} taxonomy Taxonomy name.
- *
- * @return {Array} Categories list.
- */
-export function getTerms( state, taxonomy ) {
```

Comment:
I guess we should probably leave the code for two releases and add a "deprecated" call to theses unused selectors.

Reply:
PR updated to use `deprecated`.